tests/foreman/ui/test_errata.py

```diff
@@ -443,3 +474,133 @@ def test_positive_incremental_update(self):
 
         @CaseLevel: System
         """
+
+
+@run_in_one_thread
+class FilteredErrataTestCase(UITestCase):
+    """UI Tests for filtering functionality of errata management feature"""
+
+    @classmethod
+    def set_session_org(cls):
+        """Create an organization for tests, which will be selected
+        automatically"""
+        cls.session_org = entities.Organization().create()
+
+    @tier3
+    def test_positive_errata_status_installable_param(self):
+        """Filter errata for specific content view and verify that host that
+        was registered using that content view has different states in
+        correspondence to filtered errata and `errata status installable`
+        settings flag value
+
+        @id: ed94cf34-b8b9-4411-8edc-5e210ea6af4f
+
+        @Steps:
+
+        1. Prepare setup: Create Lifecycle Environment, Content View,
+        Activation Key and all necessary repos
+        2. Register Content Host using created data
+        3. Create necessary Content View Filter and Rule for repository errata
+        4. Publish and Promote Content View to a new version and remove old
+        ones.
+        5. Go to created Host page and check its properties
+        6. Change 'errata status installable' flag in the settings and check
+        host properties once more
+
+        @Assert: Check that 'errata status installable' flag works as intended
+
+        @BZ: 1368254
+
+        @CaseLevel: System
+        """
+        env = entities.LifecycleEnvironment(
+            organization=self.session_org).create()
+        content_view = entities.ContentView(
+            organization=self.session_org).create()
+        activation_key = entities.ActivationKey(
+            environment=env,
+            organization=self.session_org,
+        ).create()
+        setup_org_for_a_rh_repo({
+            'product': PRDS['rhel'],
+            'repository-set': REPOSET['rhst7'],
+            'repository': REPOS['rhst7']['name'],
+            'organization-id': self.session_org.id,
+            'content-view-id': content_view.id,
+            'lifecycle-environment-id': env.id,
+            'activationkey-id': activation_key.id,
+        })
+        custom_entitites = setup_org_for_a_custom_repo({
+            'url': CUSTOM_REPO_URL,
+            'organization-id': self.session_org.id,
+            'content-view-id': content_view.id,
+            'lifecycle-environment-id': env.id,
+            'activationkey-id': activation_key.id,
+        })
+        with VirtualMachine(distro=DISTRO_RHEL7) as client:
+            client.install_katello_ca()
+            result = client.register_contenthost(
+                self.session_org.label,
+                activation_key.name,
+            )
+            self.assertEqual(result.return_code, 0)
+            client.enable_repo(REPOS['rhst7']['id'])
+            client.install_katello_agent()
+            client.run('yum install -y {0}'.format(FAKE_1_CUSTOM_PACKAGE))
```

Comment:
Basically, don't mind that change at all, but that PR is mostly cherry-pick from master, so don't want to have different code